sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -322,11 +322,24 @@ private[ui] class ExecutionPagedTable(
                 s"&$executionTag.pageSize=$pageSize" +
                 s"#$tableHeaderId")
 
-            <th>
-              <a href={headerLink}>
-                {header}
-              </a>
-            </th>
+            if (header == "Duration") {
+              <th>
+                <a href={headerLink}>
```

Comment:
Same comment as previously. You can avoid duplicating the th and a elements.

Reply:
ok @srowen , i will try to avoid duplicating the th and a elements.

Reply:
@srowen , done ,please review